### PR TITLE
📖 Add metal3-dev-tools integration test how to use notes

### PR DESCRIPTION
**Pull request by furkatgofurov7**
Related to https://gerrit.nordix.org/c/infra/cicd/+/12263/1/jjb/metal3/job_metal3_dev_tools_integration_test_centos.yml & https://gerrit.nordix.org/c/infra/cicd/+/12263/1/jjb/metal3/job_metal3_dev_tools_integration_test_ubuntu.yml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,27 @@ Then for all repositories :
...
+This repository is mainly used for keeping scripts related to building node and
+host images (OS). Once, new changes are pushed related to node images, there is
+no way to test if those newly build images are going to work or not when running
+integration tests in metal3-dev-env repository.
+For that reason, and also to detect any possible issues early enough, we can run
+metal3-dev-tools integration tests on a PR in this repository which:
+
+1. builds temporary node images from the changes on the PR;
+1. uses those created node images in the integration tests instead of pulling
+   them from the artifactory.
```
Comment (Sunnatillo) [inline README.md:97]: Actually, temporary image is pushed to artifactory and then downloaded back from artifactory.

Comment (furkatgofurov7) [reply README.md]: OKay thanks, forgot that part. Updated, take a look

**Review by Sunnatillo — APPROVED** [review]
/lgtm

**Pull request merged**